### PR TITLE
ci: Add Silkworm-driven consensus tests

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Review by yperbasis — APPROVED** [review]

File: circle.yml [hunk 1]
```diff
@@ -173,6 +231,24 @@ jobs:
...
+    environment:
+      TESTS_REV: v7.0.0
+    executor: linux-gcc-latest
+    steps:
+      - build_silkworm
+      - attach_workspace:
+          at: ~/package
+      - run:
+          name: "Unpack evmone"
+          background: true
```
Comment (axic) [inline circle.yml:244]: So this runs parallel? Does it really take that much time to risk any potential wrong order execution here?

File: circle.yml [hunk 2]
```diff
@@ -15,6 +15,64 @@ executors:
...
+          command: |
+            [ "$SILKWORM_BUILD" = true ] || exit 0
+            cmake --build build --parallel --target consensus
+      - save_cache:
+          name: "Save Silkworm cache"
+          key: *silkworm-cache-key
+          paths:
+            - ~/silkworm/consensus
+
+  download_consensus_tests:
```
Comment (axic) [inline circle.yml:64]: Why not chace this too?

Comment (chfast) [reply circle.yml:64]: Cache means you have to download and unpack it, just from different server.

Comment (axic) [reply circle.yml:64]: Ah ok, I thought it is some better integration with circleci.

**Review by axic — APPROVED** [review]

**Pull request merged**